tools/RealmBrowser/RealmBrowser/Classes/RLMRealmNode.m

```diff
@@ -50,14 +50,15 @@ - (BOOL)connect:(NSError **)error
                              dynamic:YES
                               schema:nil
                                error:error];
-    if (*error) {
+
+    if (error && *error) {
         NSLog(@"Realm was opened with error: %@", *error);
     }
     else {
         _topLevelClasses = [self constructTopLevelClasses];    
     }
     
-    return error != nil;
+    return !(error && *error != nil);
```

Comment:
This is incorrect in the case a user passes in nil. You need to pass in our own variable into realmWithPath and then set the user error after